### ISSUE TITLE
Avoid specifying font sizes in px

### ISSUE THREAD

**Issue by osma** [NatLibFi/Skosmos#1320]
This accessibility problem was pointed out by @kinow in [this discussion](https://github.com/NatLibFi/Skosmos/pull/1319#discussion_r864732515). Users with poor eyesight often set their browser to use a larger than normal font size and web sites should respect that. (There are other mechanisms for this as well, for example page zoom and minimum font size, which should be working already; this issue is specific to the default font size setting.)

## At which URL did you encounter the problem?

Any Skosmos page.

## What steps will reproduce the problem?
1. Set the browser font size to larger than default (e.g. Chromium: Settings -> Appearance -> Font Size -> Huge).
2. Open any Skosmos page, e.g. https://dev.finto.fi

## What is the expected output? What do you see instead?

Expected that the fonts should be much larger than normal. But they are the same as usual, ignoring the Font Size setting.

The problem is that font sizes have been declared with explicit pixel sizes such as `14px` and `16px` in CSS. Instead, relative values using `em` or `rem` could be used, which would allow the text size to scale according to the browser setting.

Compare with https://ai.finto.fi which uses a similar layout as Finto/Skosmos, but uses different font size declarations. Here I've chosen huge font size in Chromium. It's not super pretty, but at least the text is very large as it should be:

![image](https://user-images.githubusercontent.com/1132830/166882148-f1bedc80-f120-4b79-8f65-6f5041e2a76e.png)

## What browser did you use? (eg. Firefox, Chrome, Safari, Internet explorer)

Chromium (but same on Firefox)
